src/style.css

```diff
@@ -0,0 +1,157 @@
+html {
+  font-family: "Avenir", sans-serif;
+  margin: 0 auto;
+  background-color: #ffff;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.searchBar {
+  position: relative;
+  margin-top: 20px;
+  display: flex;
+  width: 100%;
+}
+/*
+.Button {
+  display: block;
+  position: absolute;
+  margin-left: 26px;
+  margin-top: 25px;
+  width: 19px;
+  height: 19px;
+} */
+
+.searchImputbig {
+  height: 70px;
+  width: 100%;
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 62px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
+  font-size: 16px;
+  font-style: inherit;
+}
+
+.searchImputsmall {
+  height: 42px;
+  width: 100%;
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 32px;
+  line-height: 11px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  font-size: 14px;
+  font-style: inherit;
+}
+
+.searchImputbig:hover {
+  width: 100%;
+  background: white;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
```

Comment:
remove outline for inputs
```
outline: none
```
![Screenshot from 2023-03-25 17-46-44](https://user-images.githubusercontent.com/33029864/227727881-95f2816d-399a-4c09-a6ff-e164315aa270.png)